src/components/NavigationConfirmationModal/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal, Button, Alert } from 'patternfly-react'
+import { msg } from '../../intl'
+
+const NavigationConfirmationModal = ({ show, onYes, onNo }) => {
+  const idPrefix = 'close-dialog-confim'
+
+  return (
+    <Modal show={show}>
+      <Modal.Header>
+        <Modal.Title id={`${idPrefix}-title`}>{msg.unsavedChangesTitle()}</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Alert type='warning' id={`${idPrefix}-body-text`}>{msg.unsavedChangesConfirmMessage()}</Alert>
```

Comment:
Right now the navigation prompt text is defined here.  This should be ok since **VmDialog/index.js** is currently the only place that uses `Prompt`.

If we were to add additional use of react-router `Prompt`, the specific message displayed would be better off defined with the `Prompt` itself and passed through here.  See [VmDialog/index.js:L499](https://github.com/oVirt/ovirt-web-ui/blob/master/src/components/VmDialog/index.js#L499)